game/script-jokes.rpy

```diff
@@ -343,6 +343,26 @@ init python in jn_jokes:
         label="joke_cheesy_pizza",
         joke_category=JNJokeCategories.bad
     ))
+    __registerJoke(JNJoke(
+        label="joke_veggie_mood",
+        joke_category=JNJokeCategories.neutral
+    ))
+    __registerJoke(JNJoke(
+        label="joke_scarecrow_award",
+        joke_category=JNJokeCategories.corny
+    ))
+    __registerJoke(JNJoke(
+        label="joke_sundae_school",
+        joke_category=JNJokeCategories.funny
+    ))
+    __registerJoke(JNJoke(
+        label="joke_burned_tongue",
+        joke_category=JNJokeCategories.neutral
+    ))
+    __registerJoke(JNJoke(
+        label="joke_pointless_pencil",
+        joke_category=JNJokeCategories.funny
+    ))
```

Comment:
Good categorization! The content matches how I'd imagine Nat reacting to each joke - good work!